src/Room.js

```diff
@@ -319,6 +320,18 @@ function notificationCallback (data) {
     return this.kuzzle.emitEvent('jwtTokenExpired');
   }
 
+  if (data.controller === 'document') {
+    data.type = 'document';
+    data.document = new Document(this.collection, data.result._id, data.result._source);
+    delete data.result;
+  }
+
+  if (data.controller === 'realtime') {
+    data.type = 'user';
+    data.user = {count: data.result.count};
```

Comment:
Does not seem to match the PR description.

> The result field is removed, replaced by a count property. 

Here, the `result` field is replaced by a `user` property, seemingly containing a `count` property.
Am I next to the plaque?